### ISSUE TITLE
Removal of Mee6 as response to NFT project.

### ISSUE THREAD

**Issue by picapi** [CyberDiscovery/meta#397]
Mee6, a Discord Bot utilised in the server has recently announced an [NFT Project](https://mee6.xyz/nft) under the brand.
The expansion plan includes the idea of creating a marketplace with every community being able to have a token that can then be traded in a decentralised way, presumably on the Ethereum Blockchain.

I feel the idea of community tokens that cannot be controlled effectively would be detrimental to the quality of discussion on the server, as seen by previous spam for on-server currencies. Personally, I also disagree with the principal that everything should be economised, but this is less relevant.

I also do not wish to implicitly support organisations doing this, and their presence in the server implicitly pads their stats and advertises the bot positively to other members, something that I feel should be avoided when dealing with NFT related companies and the environmental, social and other issues attached (https://www.theverge.com/2021/3/15/22328203/nft-cryptoart-ethereum-blockchain-climate-change, www.howtogeek.com/774410/heres-the-problem-with-nfts/,  https://www.theatlantic.com/ideas/archive/2021/04/nfts-werent-supposed-end-like/618488/ for just a few examples).

I would therefore like to propose that Mee6 is removed from the server, potentially being replaced with an alternative. This need not be a matter of urgency, but I would feel more comfortable with an alternative that doesn't explicitly participate in this fraught space.

I understand that the presence of an XP Leaderboard may be a draw for people to the server, and am unsure how portable that data may be, so would also like to hear from people how important they find that function.

**Comment by RealJammy** [comment]
I strongly agree with picapi, correlations to NFTs should never be permitted in this skid community.
